platform-modules/batch-models/src/main/scala/org/ekstep/analytics/job/report/StateAdminGeoReportJob.scala

```diff
@@ -2,24 +2,25 @@ package org.ekstep.analytics.job.report
 
 import org.apache.spark.SparkContext
 import org.apache.spark.sql._
-import org.apache.spark.sql.functions.{ col, lit, _ }
+import org.apache.spark.sql.functions.{col, _}
 import org.ekstep.analytics.framework._
-import org.ekstep.analytics.framework.util.{ JSONUtils, JobLogger }
-import org.ekstep.analytics.util.HDFSFileUtils
+import org.ekstep.analytics.framework.util.{JSONUtils, JobLogger}
+import org.ekstep.analytics.util.{HDFSFileUtils, StateAdminReportHelper}
```

Comment:
StateAdminReportHelper cannot be in the org.ekstep.analytics.util package. Create it as a trait and with the same package and add it to the  main object like
StateAdminGeoReportJob ... with StateAdminReportHelper

Reply:
Done as suggested.